staking/types/validator.go

```diff
@@ -327,6 +336,12 @@ func (w *ValidatorWrapper) SanityCheck(
 				"have %s want %s", w.Delegations[0].Amount.String(), w.Validator.MinSelfDelegation,
 			)
 		}
+		if w.Delegations[0].Amount.Cmp(minimumStake) < 0 {
```

Comment:
just check the MinSelfDelegation is >= 10000ONE, it is checking against 1 ONE now. inside Validator.SanityCheck.

Reply:
updated.